<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-3UIEUVU9" failures="1" tests="2" name="Cucumber Runner" time="45.458" errors="0" timestamp="2022-03-24T15:04:48 EDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="26.19"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.704">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Lead  found in lead list">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Lead  found in lead list
at zoho.managers.WebDriverManager.logFailure(WebDriverManager.java:66)
at pages.LeadDetailPage.validateLeadNotPresent(LeadDetailPage.java:30)
at zoho.teststeps.Leads.verifyLeadCreation(Leads.java:103)
at âœ½.Lead 'John Alex' should 'not be present' inside the grid(file:///D:/LearningJavaSelenium/CucumberLearning/src/test/resources/zoho/leads.feature:50)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Cucumber Runner -->
